packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -1,52 +1,87 @@
+  let newDate = new Date(HippaDate);
+  const time = newDate.toLocaleDateString();
+  const minutes = newDate.toLocaleTimeString();
+  const finalDate = time + ' – ' + minutes;
 
   return (
     <div className="nomination-banner-container">
       <div className="row">
         <div className="column column-10 icon-container">
-          <img src="/ksflogo.png" alt="other"/>
+          <img src="/ksflogo.png" alt="other" />
         </div>
 
         <div className="column column-80 nomination-details">
-
           <div className="row banner-top">
             <div className="column name">
               <p>Application</p>
-              <span><h1 className="nom-name"><strong>{nominationName}</strong></h1></span>
+              <span>
+                <h1 className="nom-name">
+                  <strong>{nominationName}</strong>
+                </h1>
+              </span>
             </div>
           </div>
 
           <div className="row">
             <div className="column hp-name">
               <p className="secondary-dark">HP Name</p>
-              <span><h2 className="body-font"><strong>{nomination.providerName}</strong></h2></span>
+              <span>
+                <h2 className="body-font">
+                  <strong>{nomination.providerName}</strong>
+                </h2>
+              </span>
             </div>
             <div className="column fam-name">
               <p className="secondary-dark">Family Member Name</p>
-              <span><h2 className="body-font"><strong>{nomination.representativeName}</strong></h2></span>
+              <span>
+                <h2 className="body-font">
+                  <strong>{nomination.representativeName}</strong>
+                </h2>
+              </span>
             </div>
             <div className="column created-at">
               <p className="secondary-dark">Submission Date</p>
-              <span><h2 className="body-font"><strong>{date}</strong></h2></span>
+              <span>
+                <h2 className="body-font">
+                  <strong>{date}</strong>
+                </h2>
+              </span>
             </div>
             <div className="column amount">
-               <p className="secondary-dark">Grant Amount Requested</p>
-              <span><h2 className="body-font"><strong>{formattedAmount ? `$${formattedAmount}` : ''}</strong></h2></span>
+              <p className="secondary-dark">Grant Amount Requested</p>
+              <span>
+                <h2 className="body-font">
+                  <strong>{formattedAmount ? `$${formattedAmount}` : ''}</strong>
+                </h2>
+              </span>
+            </div>
+            <div className="column hippa">
+              <p className="secondary-dark">HIPPA Date</p>
+              <span>
+                <h2 className="body-font">
+                  <strong>{finalDate != '12/31/1969 – 6:00:00 PM' ? finalDate : 'Awaiting HIPPA'}</strong>
```

Comment:
Likely not a big deal in this case, but we should (almost) always use `!==` instead of `!=` as best practice.